unstructured/documents/email_elements.py

```diff
@@ -24,20 +24,13 @@ def __init__(
         self,
         name: str,
         text: str,
-        datestamp: Union[datetime, NoDatestamp] = NoDatestamp(),
-        element_id: Union[str, uuid.UUID, NoID, UUID] = NoID(),
+        datestamp: datetime | NoDatestamp = NoDatestamp(),
+        element_id: str | NoID = NoID(),
     ):
         self.name: str = name
         self.text: str = text
```

Comment:
I think this line goes away because `Text` takes care of adding this attribute (self.text) and setting its value when you call `super()` below.